packages/block-editor/src/components/block-preview/auto.js

```diff
@@ -110,6 +115,12 @@ function AutoBlockPreview( {
 					} }
 				>
 					{ contentResizeListener }
+					{ duotonePresets.map( ( preset ) => (
+						<PresetDuotoneFilter
+							preset={ preset }
+							key={ preset.slug }
+						/>
+					) ) }
```

Comment:
And can we call these `svgFilters` like in https://github.com/WordPress/gutenberg/pull/37727/files#diff-ee5edd4222091b38cbbcb2fe70a50dba404f2724891346f82fe5d591623cfd54R187?

I've been trying to avoid calling things "duotone" outside of the duotone block supports to encourage people to think more broadly about how SVGs and filters can be used to create unique designs.

Some duotone specific stuff is unavoidable at this point—it's fine for now to import and use `PresetDuotoneFilter` directly, but the longer term vision is using a filter that extenders can hook into. https://github.com/WordPress/gutenberg/pull/37727#discussion_r840444001